enginerpc/engine_impl.cc

```diff
@@ -517,8 +525,11 @@ grpc::Status engine_impl::AddHostComment(grpc::ServerContext* context
     auto it = host::hosts.find(request->host_name());
     if (it != host::hosts.end())
       temp_host = it->second;
-    if (temp_host == nullptr)
+    if (temp_host == nullptr) {
+      logger(log_runtime_warning, basic)
+        << "Warning:  AddHostComment could not found a host name!";
```

Comment:
here we have a detected error. It is not a good thing to log it in the centengine logs. I think it would be better to send an error directly to the user via the script return.